docker-compose.yml

```
@@ -0,0 +1,18 @@
+version: "3.9"
+   
+services:
+  db:
+    image: postgres
+    environment:
+      - POSTGRES_DB=mathesar
+      - POSTGRES_USER=mathesar
+      - POSTGRES_PASSWORD=mathesar
```

```suggestion
      - POSTGRES_PASSWORD=mathesar
    ports:
      - "5432:5432"
```
This will let psql see the DB.

---

I think we should also add a volume for data persistence.  I usually prefer one in some out-of-the-way place in the repo, and `.gitignored`, since it makes it easy to manually intervene if you want.

---

Done